src/hashtable.c

```diff
@@ -669,16 +672,50 @@ static int expand(hashtable *ht, size_t size, int *malloc_failed) {
     return resize(ht, size, malloc_failed);
 }
 
+static inline int checkCandidateInBucket(hashtable *ht, bucket *b, int pos, const void *key, int *pos_in_bucket, int *table_index, int table) {
```

Comment:
Let's put the return-by-pointer variables `pos_in_bucket` and `table_index` last. It's an informal convention.

~~Here is a suggestion for a comment to explain how use the function.~~

```suggestion
static inline int checkCandidateInBucket(hashtable *ht, bucket *b, int pos, const void *key, int table, int *pos_in_bucket, int *table_index) {
```

Reply:
Ah, you added comments. :) Ignore my suggested comment.